content/docs/beans/_index.md

```
@@ -1,41 +1,301 @@
+{{< image-sh src="02-add-beans-button.png" text="Add new Beans button" >}}
+
+4. In the **Details** form at the right, fill bean **name** and Java **type**:
+   - **name** → `bookDataSource`
+   - **type** → `org.postgresql.ds.PGSimpleDataSource`
+{{< image-sh src="03-bean-name-type.png" text="Bean name and type" >}}
+
+5. Lower in the **Details** form, add the connection details under **properties**. Click **Add a new property** button:
+{{< image-sh src="04-bean-properties.png" text="Add Bean properties" >}}
+
+6. Fill properties name and value as following:
+   | Key | Value |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)